packages/app/src/App.tsx

```diff
@@ -59,18 +62,20 @@ const App: FC<{}> = () => {
     toggleTheme,
   };
   return (
-    <ThemeContext.Provider value={themeContext}>
-      <ThemeProvider theme={backstageTheme}>
-        <CssBaseline>
-          <AlertDisplay forwarder={alertApiForwarder} />
-          <Router>
-            <Root>
-              <AppComponent />
-            </Root>
-          </Router>
-        </CssBaseline>
-      </ThemeProvider>
-    </ThemeContext.Provider>
+    <AppProvider>
```

Comment:
could have been fancy with `<app.Provider>...<app.RootComponent>` and using getters :)